chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -63,6 +63,12 @@ if [ $(id -u) -gt 0 ]; then
     exit 1
 fi
 
+if test -n "$SSH_CONNECTION" -a -z "$STY"; then
```

Comment:
Hmm, it seems there are roughly an equal number of usages of `if test -...` and `if [ ]` / `if [[ ]]`.  I wish we could be consistent about this :-/  My take is that `test` is the more old-fashioned style, from the POSIX era where it actually forked a separate process.  Personally I like the conditional to be wrapped in parentheses, especially when it spans multiple lines.